source/dem/CMakeLists.txt

```diff
@@ -90,7 +90,7 @@ add_library(lethe-dem
   ../../include/dem/particle_wall_linear_force.h
   ../../include/dem/particle_wall_nonlinear_force.h
   ../../include/dem/periodic_boundaries_manipulator.h
-  ../../include/dem/post_processing.h
+        ../../include/dem/dem_post_processing.h
```

Comment:
Can you put in alphabetical order please

Reply:
Sorry about that, I never saw the file because I use the refactor tools in CLion and forget about that detail